### PR TITLE
Add compilation of Bend internal inet to HVM

### PR DESCRIPTION
**Pull request by developedby**

### CODE REVIEW

File: Bend/Compile/NetToHvm/get-redexes.agda [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+open import Data.Bits.to-nat
+open import Bend.Net.Type
+open import Bend.Net.Node.Type
+open import Bend.Net.Port.Type
+
+-- Returns the redexes of the net as pairs of node indices
+get-redexes : Net → List (Pair Nat Nat)
+get-redexes net =
+  let redexes = check-nodes (to-list (Net.nodes net)) map-new in
+  let (_ , redexes) = unzip (to-list redexes) in
```
Comment (enricozb) [inline Bend/Compile/NetToHvm/get-redexes.agda:27]: we should probably have a `Data.Map.values`

File: Bend/Fun/Term/dsl.agda [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+-- Numbers
+#u : Nat → Term
+#u n = Num (U24 n)
+
+#i : Int → Term
+#i n = Num (I24 n)
+
+#f : Float → Term
+#f n = Num (F24 n)
```
Comment (enricozb) [inline Bend/Fun/Term/dsl.agda:61]: good notation, might steal this for imp

File: check_all.sh [hunk 3]
```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
 
-find . -name "*.agda" -not -path "*.tmp*" | while read -r file; do
+find Bend/ -name "*.agda" -not -path "*.tmp*" | while read -r file; do
```
Comment (enricozb) [inline check_all.sh:3]: this probably shouldn't be committed right?

Comment (developedby) [reply check_all.sh:3]:
```suggestion
find . -name "*.agda" -not -path "*.tmp*" | while read -r file; do
```
Oops

File: check_all.sh [hunk 4]
```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
 
-find . -name "*.agda" -not -path "*.tmp*" | while read -r file; do
+find $1 -name "*.agda" -not -path "*.tmp*" | while read -r file; do
```
Comment (edusporto) [inline check_all.sh:3]: Maybe do find on `.` if `$1` is not set, running just `./check_all.sh` fails with
```
find: illegal option -- n
usage: find [-H | -L | -P] [-EXdsx] [-f path] path ... [expression]
       find [-H | -L | -P] [-EXdsx] -f path [path ...] [expression]
```

**Review by edusporto — APPROVED** [review]

File: check_all.sh (hunk 4 above)
Comment (developedby) [reply check_all.sh:3]: I wonder if this happens depending on the shell, for me it works as expected.

**Review by enricozb — APPROVED** [review]

**Pull request merged**